app/controllers/users_controller.rb

```diff
@@ -0,0 +1,72 @@
+class UsersController < ApplicationController
+  before_action :set_user, only: [:show, :edit, :update, :destroy]
+
+  # GET /users
+  # GET /users.json
+  def index
+    @users = User.order(:forename)
+  end
+
+  # GET /users/1
+  # GET /users/1.json
+  def show; end
+
+  # GET /users/new
+  def new
```

Comment:
Is this needed? I see a `create` function below that looks more useful

Reply:
Most likely, no. It was auto-generated by rails and I never removed it.

Reply:
No worries for now, things like this can be cleaned up down the line